EventFilter/SiPixelRawToDigi/plugins/SiPixelRawToDigiGPU.cc

```diff
@@ -373,19 +420,59 @@ SiPixelRawToDigiGPU::produce( edm::Event& ev, const edm::EventSetup& es)
     break;
   }
 
+  int32_t nclus=-1;
+  std::vector<AccretionCluster> aclusters(256);
```

Comment:
this is a clear time-waster... to be optimized (if we care this gpu2cpu part...

Reply:
> if we care this gpu2cpu part

I think for that we care mostly about correctness, not much about performance, since it's not the intended workflow